packages/components/radio/src/radio.css

```diff
@@ -0,0 +1,87 @@
+/* General........................................................ */
+
+:host {
+	& label {
+		display: block;
+		position: relative;
+		padding-left: var(--ts-unit-plus);
+		margin: var(--ts-unit-half);
+		cursor: pointer;
+		font-size: var(--ts-fontsize);
+		-webkit-user-select: none;
+		-moz-user-select: none;
+		-ms-user-select: none;
+		user-select: none;
+
+		&:hover {
+			& span {
+				background-color: var(--ts-color-gray-lighter);
+			}
+		}
+
+		& input {
+			position: absolute;
+			opacity: 0;
+			cursor: pointer;
+			height: 0;
+			width: 0;
+		}
+
+		& span {
+			box-shadow: var(--ts-radio-unselected-boxshadow-border-width) var(--ts-color-gray-light);
+			position: absolute;
+			top: 0;
+			left: 0;
+			height: var(--ts-unit);
+			width: var(--ts-unit);
+			border-radius: 50%;
+
+			&:after {
+				content: '';
+				position: absolute;
+				display: none;
+				top: var(--ts-unit-quarter);
+				left: var(--ts-unit-quarter);
+				width: 8px;
```

Comment:
Please use variable and maybe some division of unit

Reply:
Done